src/pure.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ReactElement, ReactNode, Suspense } from 'react'
+import { act, create, ReactTestRenderer } from 'react-test-renderer'
+import asyncUtils from './asyncUtils'
+import { cleanup, addCleanup, removeCleanup } from './cleanup'
+
+// TODO: Add better type, currently file is utilizing minimum types to work
+// TODO: Attempt to refactor code to remove ESLint disables if possible
```

Comment:
We probably don't need these TODO lines anymore?

Reply:
We do not, please remove them.